README.md

```diff
@@ -108,6 +108,7 @@ $ m2cgen <pickle_file> --language <language> [--indent <indent>] [--class_name <
          [--module_name <module_name>] [--package_name <package_name>] [--namespace <namespace>]
          [--recursion-limit <recursion_limit>]
 ```
+Don't forget that for unpickling serialized model objects their classes must be defined in the top level of an importable module in the unpickling environment.
```

Comment:
Wordings are taken from the official documentation: https://docs.python.org/3/library/pickle.html#what-can-be-pickled-and-unpickled.